src/java/test/org/apache/zookeeper/server/CreateTTLTest.java

```diff
@@ -69,6 +75,23 @@ public void testCreate()
     }
 
     @Test
+    public void testBadTTLs()
+            throws IOException, KeeperException, InterruptedException {
+        Stat stat = new Stat();
```

Comment:
Ah! the first version of this went through the zk client-ish interface, so probably not needed anymore